File: snippets/csharp/FiatCurrencies.cs
using Breez.Sdk;

public class FiatCurrenciesSnippets
{
    public void ListFiatCurrencies(BlockingBreezServices sdk)
    {
        // ANCHOR: list-fiat-currencies
        try
        {
            var fiatCurrencies = sdk.ListFiatCurrencies();
        }
        catch (Exception)
        {
            // Handle error
        }
        // ANCHOR_END: list-fiat-currencies
    }

    public void FetchFiatRates(BlockingBreezServices sdk)
    {
        // ANCHOR: fetch-fiat-rates
        try
        {
            var fiatRates = sdk.FetchFiatRates();
        }
        catch (Exception)
        {
            // Handle error
        }
        // ANCHOR_END: fetch-fiat-rates
    }
}

File: snippets/go/fiat_currencies.go
package example

import (
	"log"
)

func ListFiatCurrencies() error {
	// ANCHOR: list-fiat-currencies
	fiatCurrencies, err := sdk.ListFiatCurrencies()
	if err != nil {
		return err
	}
	log.Printf("%#v", fiatCurrencies)
	// ANCHOR_END: list-fiat-currencies
	return nil
}

func FetchFiatRates() error {
	// ANCHOR: fetch-fiat-rates
	fiatRates, err := sdk.FetchFiatRates()
	if err != nil {
		return err
	}
	log.Printf("%#v", fiatRates)
	// ANCHOR_END: fetch-fiat-rates
	return nil
}
